contrib/cirrus/lib.sh

```diff
@@ -298,8 +309,6 @@ install_conmon(){
 }
 
 install_criu(){
```

Comment:
Since the only invocation (that I can find) of `install_criu` was removed (fedora_setup.sh), should the function be removed altogether?

Reply:
Maybe in a future PR, though.

Reply:
My eventual goal is to allow either RPM or source based dependency installs.  Even if not, yes it can be removed in a future PR if need be.